src/config.rs

```diff
@@ -15,12 +15,27 @@
 ///
 /// If there's a config option you want to see or think is missing, please open an issue!
 pub struct Config<S: ToString> {
-    /// Sets the scale of the buffer. The buffer will automatically scale to the size of the
-    /// window. By default this will be the same size as the window_size.
+    /// Sets the pixel dimensions of the buffer. The buffer will automatically scale to the size of
+    /// the window. By default this will be the same as the window_size.
     pub buffer_size: (u32, u32),
+    /// If this is true, the window created by mini_gl_fb will be set to resizable. This can be
+    /// changed later.
     pub resizable: bool,
     pub window_title: S,
     pub window_size: (f64, f64),
+    pub invert_y: bool
+}
+
+impl<S: ToString + Clone> Clone for Config<S> {

```

Comment:
> Hmm is the ToString constraint breaking derive(Clone)?
Yes, because S does not implement Clone. :P 

> If ToString is an issue it can be removed from the definition of Config, but left as a constraint elsewhere where Config is consumed.
ToString should remain a constraint so that people can't set the window title to a File handle. Even if no functions will accept that...